contracts/token/erc20/Erc20.sol

```diff
@@ -88,6 +91,10 @@ contract Erc20 is IErc20 {
 
     /// @inheritdoc IErc20
     function decreaseAllowance(address spender, uint256 subtractedValue) external virtual override returns (bool) {
+        uint256 currentAllowance = allowances[msg.sender][spender];
+        if (currentAllowance >= subtractedValue) {
```

Comment:
Should have been `currentAllowance < subtractedValue`.

Reply:
I agree, should have been currentAllowance < subtractedValue.